src/Microsoft.ApplicationInsights/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -44,9 +44,17 @@ public PlatformImplementation()
         /// </summary>
         public string ReadConfigurationXml()
         {
-            // Config file should be in the base directory of the app domain
-            string configFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "ApplicationInsights.config");
-
+            string configFilePath;
+            try
+            {
+                // Config file should be in the base directory of the app domain
+                configFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "ApplicationInsights.config");
```

Comment:
can you instead move this line to the already existing try..catch blocks?

Reply:
Not elegantly as the `configFilePath` is referenced in the other exceptions.